File: src/main/java/cz/krystofcejchan/main/after_startup/DeletingAllSlashCommandsFromAllGuilds.java
package cz.krystofcejchan.main.after_startup;

import cz.krystofcejchan.main.Main;
import net.dv8tion.jda.api.entities.Guild;

public class DeletingAllSlashCommandsFromAllGuilds implements IAfterStartUp {
    @Override
    public void doAfterStartUp() {
        Main.publicJDA.getGuilds().forEach(Guild::updateCommands);
    }
}

File: src/main/java/main/afterStartUp/DeletingAllSlashCommandsFromAllGuilds.kt
package main.afterStartUp

import main.StartUp

class DeletingAllSlashCommandsFromAllGuilds : IAfterStartUp {
    override fun doAfterStartUp() {
        // SlashCommandManager slashCommandManager = new SlashCommandManager();
        for (guild in StartUp.publicJDA.guilds) {
            guild.updateCommands()
        }
    }
}